plugins/databases/access/src/main/java/org/apache/hop/pipeline/transforms/accessoutput/AccessOutput.java

```diff
@@ -323,13 +323,12 @@ public List<ColumnBuilder> prepareTableColumns(IRowMeta row) {
           column.setLength(DataType.SHORT_DATE_TIME.getFixedSize());
           break;
         case IValueMeta.TYPE_STRING:
-          if (length < 255) {
+          if (length < DataType.TEXT.getMaxSize()/DataType.TEXT.getUnitSize() ) {
```

Comment:
What is this division supposed to do?
A bit annoying that the Jackcess documentation doesn't have much info.